about.html

```diff
@@ -0,0 +1,128 @@
+        invitation list including organizations and individuals</p>
+    </section>
+    <section class="about-intrduction-detail desktop-element black center">
+      <p> The Global Summit brings together the community of experts, acadamics and activists who comprise the
+        creative commons affiliate network in a differnt country every two. This yea, we hope to expand our
+        invitation list including organizations and individuals who want to work with us on shared projects that advance
+        the cause of the commons, free culture and open knowledge . So if you're active and engaged in the worlds of
+        open content and knowledge - free software advocates, Wikipedians, Open, Knowledge galleries, libraries,
+        museums, archives, governmentes and foundations, lawyers and activists - we hope you'll consider joining us this
+        year to build a stronger, more vibrant commons together.</p>
+    </section>
+    <section class="event-email about-event-email ">
+      <p class="black event-notification lato ">please contact us per Email for any further questions about CC Global
+        Summit 2022!
+        <br> <span>aleazaryilma@gmail.com</span>
+      </p>
+
+    </section>
+    <section class="about-logo lato">
+      <header class="black mobile_element">Creative Commons <br> Global Summit 2015 Logo</header>
+      <header class="black about-log-title desktop-element">Creative Commons Global Summit 2022 Logo</header>
+      <p class="black mobile_element">The logo of CC Global summit 2015 was decided through the logo competition from 8
+        june to 7
+        july
+      </p>
+      <p class="desktop-element about-log-detail">
+        The logo of CC Global Summit 2015 was decided through the logo competition from 8 june to 7 july. After online
+        public votiong and selection committee's judging process, the logo from Naresh Agarawai won the prize after
+        consultation with Mr. Agrawai we optimized the prize-winning-logo for this year's summit as follwing:
+      </p>
+      <img class="about-logo-image" src="images/logo.png" alt="coffee and code logo">
+      <hr>
+      <div class="past-summits">
+        <header class="past-summits-titile">See the past CC Global Summits</header>
+        <p class="past-summits-detail black">Take a look at the last two CC Global Summits which took place in Buenos
+          Aires an din Warsaw</p>
+        <div class="summits">
+          <div class="past-summit">
+           <img src="images/previoussummitone.jpg" alt="previous summit one">
+          </div>
+          <div class="past-summit">
+            <img src="images/summittwo.jpg" alt="previous summit two">
+          </div>
+        </div>
+      </div>
+    </section>
+   
+  </main>
+  <footer class="footer about-footer">
+    <div class="footer-logo">
+    
+    </div>
+    <div class=" footer-copyright">2022 Creative Commons Korea.<br>
+      Some Rights Reserved.
+    </div>
```

Comment:
- [ ] On your about page footer, I can see that the footer is skewed to the right and not centered correctly. This is because your logo is missing on the left side of the footer. Please include the logo 👍 
![image](https://user-images.githubusercontent.com/46839548/181446442-686a7969-b7fd-4642-87ae-7e458d1e2934.png)